docs/json/radarr/cf/hbo.json

```diff
@@ -9,7 +9,7 @@
       "negate": false,
       "required": true,
       "fields": {
-        "value": "\\b(hbo)(?![ ._-]max)\\b(?=[ ._-]web[ ._-]?(dl|rip)\\b)"
```

Comment:
There was no need to do a negative lookahead on hboxmax, when there is a positive lookahead on webdl/webrip